core/src/main/java/edu/wpi/grip/core/operations/composite/FindBlobsOperation.java

```diff
@@ -44,6 +48,8 @@
   private final InputSocket<Mat> inputSocket;
   private final InputSocket<Number> minAreaSocket;
   private final InputSocket<List<Number>> circularitySocket;
+  private final InputSocket<List<Number>> convexitySocket;
+  private final InputSocket<List<Number>> inertiaRatioSocket;
   private final InputSocket<Boolean> colorSocket;
```

Comment:
Nitpick, can you put these in the same order as they are declared in the `getInputSockets` method, same thing about their declaration in the constructor and in the socket hints list.